bin/cleaner.rb

```diff
@@ -5,9 +5,16 @@
 require 'ike_artifactory'
 
 actually_delete = false
-if ARGV[0] == '--actually-delete'
-  ARGV.shift
-  actually_delete = true
+delete_empty_path = false
+while ARGV[0] =~ /^--/ do
+  arg = ARGV.shift
```

Comment:
So I think this says "if there are any arguments before `--actually-delete` and `--delete-empy-path`, throw those arguments away and ignore them"

Reply:
Oh wait, I misread this. Please ignore my comment.